src/main/java/nextstep/courses/domain/Size.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.courses.domain;
+
+public class Size {
+
+    private static final double WIDTH_RADIO = 3;
```

Comment:
'비율'이란 의미의 `RATIO`로 이름을 지어보면 좋을 것 같습니다.
```suggestion
    private static final double WIDTH_RATIO = 3;
```